### ISSUE TITLE
posters display in incorrect proportion

### ISSUE THREAD

**Issue by eightsheep67** [jellyfin/Swiftfin#1130]
### Describe the bug

some movie posters pics are not normal poster proportion then will display in incorrect proportion on version 1.1

### Application version

1.1

### Where did you install the app from?

None

### Device information

iphone 14

### OS version

ios 17.5.1

### Jellyfin server version

10.9.6

**Comment by LePips** [comment]
What is meant by "proportion"? Can I get a screenshot please?

**Comment by PhilMarlow2** [comment]
I have the same issue with actor images. The aspect ratio is wrong when the pictures are not from TheMovieDB. When the original picture is a square then the actors are thin and tall :). The sides are not clipped like in the old tvOS version.

**Comment by tobylibo** [comment]
I have the same issue. To put it simply, Swiftfin doesn't automatically trim non-standard posters and fanarts. (There are a large number of posters and fanarts that are not standard size in aspect ratios)

**Comment by MOZGIII** [comment]
The easiest way to reproduce is to set up a library that extracts the posters from the file contents as opposed to metadata:

<img width="845" alt="Image" src="https://github.com/user-attachments/assets/25337a27-7c2c-4c16-b945-8355862fbb92" />

Or, of course, just use custom videos.

---

The bug:

<details><summary>Details</summary>
<p>

![Image](https://github.com/user-attachments/assets/5da34534-8f8b-4448-a956-a7db43ee11d6)

![Image](https://github.com/user-attachments/assets/2705e0c8-4e0e-4590-ab46-77b4d4afabb2)

</p>
</details> 

How it is supposed to look:

<details><summary>Details</summary>
<p>

<img width="1066" alt="Image" src="https://github.com/user-attachments/assets/dce7eb33-3c01-4f82-92cb-38658b341280" />

</p>
</details>

**Comment by LePips** [comment]
Non-standard aspect ratio'd images will aspect fill the poster container. The sizes of the views are fixed and won't adjust to the image's dimensions which is intentional.

**Comment by MOZGIII** [comment]
My example demonstrates standard aspect ratio image displayed incorrectly. You can see the official Jellyfin client handling it right.
